item-search/README.md

```diff
@@ -180,124 +156,75 @@ searching on specific Item properties.
 
-Response with `200 OK`:
-```json
-{
-    "rel": "next",
-    "href": "http://api.cool-sat.com/search",
-    "method": "POST",
-    "body": {
-        "next": "a9f3kfbc98e29a0da23"
-    }
-}
-```
-The above link tells the client not to merge (default of false) so it is only required to pass the next token in the body.
+The STAC search endpoint, `/search`, by default only accepts a limited set of parameters to limit the results
+by properties. The Query extension adds a new parameter, `query`, that can take a number of comparison operators to
+match predicates between the fields requested and the values of Items. It can be used with both GET and POST, though
+GET includes the exact same JSON. The full details on the JSON structure are specified in the [query 
+fragment](../fragments/query/).
 
-Request to `POST /search`:
-```json
-{
-    "next": "a9f3kfbc98e29a0da23"
-}
-```
+STAC API's that support the query functionality must include the conformance class 
+<http://stacspec.org/spec/api/1.0.0-beta.1/extensions/item-search#query> in the `conformsTo` response at
+the root (`/`) landing page, to indicate to clients that they will respond properly to requests that use
+the `query` parameter on the endpoint linked to by the `search` rel.
 
-##### POST search using headers
-Request to `HTTP POST /search`:
-```json
-{
-    "bbox": [-110, 39.5, -105, 40.5],
-    "page": 2,
-    "limit": 10
-}
-```
+### Sort
 
-Response with `200 OK`:
-```json
-{
-    "type": "FeatureCollection",
-    "features": [],
-    "links": [
-        {
-            "rel": "next",
-            "href": "http://api.cool-sat.com/search",
-            "method": "POST",
-            "headers": {
-                "Search-After": "LC81530752019135LGN00"
-            }
-        }
-    ]
-}
-```
+- **Conformance URI:** <http://stacspec.org/spec/api/1.0.0-beta.1/extensions/item-search#sort>
+- **Extension [Maturity Classification](../extensions.md#extension-maturity):** Pilot
+- **Definition**: [STAC API - Sort Fragment](../fragments/sort/)
 
-This tells the client to POST to the search endpoint with the header `Search-After` to obtain the next set of results:
+By default, the STAC search endpoint `/search` returns results in no specified order. Whatever order the results are in 
+is up to the implementor, and will typically default to an arbitrary order that is fastest for the underlying data store 
+to retrieve results. This extension adds a new parameter, `sortby`, that lets a user specify a comma separated list of
+field names to sort by, with an indication of direction. It can be used with both GET and POST, the former using '+' and
+'-' to indicate sort order, and the latter including a 'direction' field in JSON. The full description of the semantics
+of this extension can be found in the [sort fragment](../fragments/sort).
 
-Request:
-```http
-POST /search
-Search-After: LC81530752019135LGN00
-```
+STAC API's that support the query functionality must include the conformance class 
```

Comment:
The conformsTo details are repeated many times. Maybe there a more clever way without writing so much? The shorter the docs, the more likely people read it. Is it enough to specofy Conformance URI at the top and have somewhere centrally defined that you need conformance classes? At a place you can't miss it. A bold note in the core or so...

Reply:
Ah, that's fair. I think for these I was going more towards the 'requirements' style, to make clear exactly what you have to do. But my plan is to do actual requirements in a next release, in their own document, and to keep this one as the narrative. So I think it's ok to just remove some of the detail/repetition.

Reply:
The list with maturity, conformance uri etc.

Reply:
We still need to document conformsTo on the landing page anyway, we have not done that yet: https://github.com/radiantearth/stac-api-spec/issues/27#issuecomment-738883830